### DISCUSSION TITLE
Automatic branch deletion feature request

### DISCUSSION THREAD

**Discussion by jonathanl-telenav** [community/community#29985] [Pull Requests]
It would be nice if there was a way to schedule automatic branch deletion for a certain number of days after the merge rather than immediately. Just a simple text box where you can type the number of days should be good.

**Comment by aayushchugh** [comment]
I think you can find some actions for that now
you can check this [action](https://github.com/marketplace/actions/delete-merged-branch) but remember to change event to schedule
